optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/PlannerBenchmarkConfig.java

```diff
@@ -435,7 +435,7 @@ public static PlannerBenchmarkConfig createFromFreemarkerXmlReader(Reader templa
      */
     public static PlannerBenchmarkConfig createFromFreemarkerXmlReader(Reader templateReader, Object model,
             ClassLoader classLoader) {
-        Configuration freemarkerConfiguration = new Configuration(Configuration.VERSION_2_3_31);
+        Configuration freemarkerConfiguration = new Configuration(Configuration.VERSION_2_3_32);
```

Comment:
FYI we can prevent some of these situations by using `new Version(2, 3, 32)` instead.

Reply:
I will do it in a separate PR as we need to merge and restart the release soon.